diff --git a/node_modules/expo-dev-launcher/expo-dev-launcher-gradle-plugin/settings.gradle.kts b/node_modules/expo-dev-launcher/expo-dev-launcher-gradle-plugin/settings.gradle.kts
new file mode 100644
index 0000000..ef7c37d
--- /dev/null
+++ b/node_modules/expo-dev-launcher/expo-dev-launcher-gradle-plugin/settings.gradle.kts
@@ -0,0 +1,7 @@
+pluginManagement {
+  repositories {
+    gradlePluginPortal()
+    google()
+    mavenCentral()
+  }
+}
