diff --git a/ios/IapSerializationUtils.swift b/ios/IapSerializationUtils.swift
--- a/ios/IapSerializationUtils.swift
+++ b/ios/IapSerializationUtils.swift
@@ -204,8 +204,8 @@
         "webOrderLineItemID": t.webOrderLineItemID
     ]

-    #if compiler(>=5.10)
-    if #available(iOS 15.2, tvOS 15.2, *) {
+    #if compiler(>=6.0)
+    if #available(iOS 18.0, tvOS 18.0, *) {
         result["appTransactionID"] = t.appTransactionID
     }
     #endif
